eva-release/src/main/java/uk/ac/ebi/eva/release/dto/ReleaseStatsPerSpeciesDto.java

```diff
@@ -32,22 +32,32 @@
 
     private Long currentRs;
 
+    private Long remappedCurrentRs;
+
     private Long multiMappedRs;
 
     private Long mergedRs;
 
+    private Long splitRs;
+
     private Long deprecatedRs;
 
     private Long mergedDeprecatedRs;
 
     private Long unmappedRs;
 
+    private Long ssClustered;
```

Comment:
I'm not sure we'll have this value at the species level.
If it is a column in the database I'm not sure we want to expose it.